packages/css/postcss.config.js

```diff
@@ -1,10 +1,29 @@
 module.exports = {
   plugins: [
     require('postcss-import'),
+    postcssApply(),
     require('postcss-nesting'),
     require('cssnano')({
       preset: 'default',
     }),
     require('autoprefixer'),
   ],
 };
+
+function postcssApply() {
+  return {
+    postcssPlugin: '@apply', // Add support for @apply directive to enable inlining utility classes
+    AtRule: {
+      apply: (rule) => {
+        rule.root().walkRules((from) => {
```

Comment:
I guess this assumes referenced rules are imported before and thus available? Good to know if we want to use this for other use-cases.

Reply:
Yes, just like tailwind, a rule needs to exist in the same code base, but import order does not matter :)